addons/core/functions/fnc_getDefaultRadioSettings.sqf

```diff
@@ -26,80 +26,74 @@
   Public: Yes
 */
 
-params [["_isSW", true, [false]], ["_unit", TFAR_currentUnit, [objNull]]];
+params [
+    ["_isSR", true, [false]],
+    ["_unit", TFAR_currentUnit, [objNull]],
+    ["_radioClass", "", [""]]
+];
 
-if (_isSW) then {
+private "_return";
```

Comment:
*puke*
alternative is to replace first `then` with exitWith and put the else block outside.
But this is not perf critical code so... meh